examples/gnrc_networking/README.md

```diff
@@ -18,13 +20,15 @@ First, create a tap interface:
 Now you can start the `gnrc_networking` example by invoking `make term`. This should automatically connect to the `tap0` interface. If
 this doesn't work for some reason, run `make` without any arguments, and then run the binary manually like so (assuming you are in the `examples/gnrc_networking` directory):
 
+    sudo bin/native/gnrc_networking.elf tap0
```

Comment:
But... The entire purpose of the text above is to instruct readers what to do if make term *doesn't* work as expected?